tests/python/unittest/test_tir_transform_hoist_expression.py

```diff
@@ -448,7 +448,8 @@ class TestHoistLetExpr(BaseBeforeAfter):
     def before(A: T.Buffer[(4, 4), "float32"]):
         for i, j in T.grid(4, 4):
             x = T.var("float32")
-            A[i, j] = tir.Let(x, T.cast(i + 1, "float32"), 5.0 * x + T.cast(j, "float32"))
+            with T.let(x, T.cast(i + 1, "float32")):
+                A[i, j] = 5.0 * x + T.cast(j, "float32")
```

Comment:
Are we changing the original let expression to TIR's let stmt?

Reply:
yes, I have recovered it to the original `Let` expr, and added the `Let` in `/script/tir/intrin.py`